.github/scripts/build-fleet-binaries.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Description: build fleet binary and image with debug flags
+
+set -euxo pipefail
+
+if [ ! -d ./cmd/fleetcontroller ]; then
+  echo "please change the current directory to the fleet repo checkout"
+  exit 1
+fi
```

Comment:
```suggestion
```

Not needed as CI is always in the right directory. I'd remove it so we don't have to maintain the path to `cmd/fleetcontroller`.